services/core-web/src/components/noticeOfWork/NOWActionWrapper.js

```diff
@@ -79,18 +85,24 @@ export class NOWActionWrapper extends Component {
       ].includes(nextProps.tab);
 
       if ((tabChanged || progressNoWExists || progressChanged) && tabShouldIncludeProgress) {
-        this.handleDisableTab(nextProps.tab, nextProps.progress);
+        this.handleDisableTab(nextProps.tab, nextProps.progress, nextProps.isDisabledReviewButton);
       }
     }
   };
 
-  handleDisableTab = (tab, progress) => {
+  handleDisableTab = (tab, progress, isDisabledReviewButton) => {
     if (tab) {
+      //application_progress_status_code does not have end_date. Status:In Progress
       if (!isEmpty(progress[tab]) && !progress[tab].end_date) {
         this.setState({ disableTab: false });
-      } else if (isEmpty(progress[tab])) {
+      }
+      //DisabledReviewButton is not true (for consultaiton and referral tabs) and
+      //application_progress_status_code does not exist. Status:Not started
```

Comment:
Can you make it more specific as to why it's not disabled? ie) "Buttons to display ref/con package deviates from the standard logic and are visible before progress is started)"